aws_signing_helper/serve.go

```diff
@@ -228,6 +270,9 @@ func Serve(port int, credentialsOptions CredentialsOpts) {
 	refreshableCred.SecretAccessKey = credentialProcessOutput.SecretAccessKey
 	refreshableCred.Token = credentialProcessOutput.SessionToken
 	refreshableCred.Expiration, _ = time.Parse(time.RFC3339, credentialProcessOutput.Expiration)
+	refreshableCred.Code = "Success"
```

Comment:
Extract "Success" and "AWS-HMAC" to constants, maybe?

Reply:
I've changed this and added another commit to this PR.